resin/pyproject.toml

```
@@ -0,0 +1,62 @@
+[build-system]
+requires = [
+    "hatchling>=0.12.0",
+    "setuptools; python_version < '3.0'",
+]
+build-backend = "hatchling.build"
+
+[project]
+name = "datadog-resin"
+description = "The Resin check"
+readme = "README.md"
+license = "BSD-3-Clause"
+keywords = [
+    "datadog",
+    "datadog agent",
+    "datadog check",
+    "resin",
+]
+authors = [
+    { name = "Datadog", email = "packages@datadoghq.com" },
```

Should we keep the contributor as the author?
```suggestion
    { name = "Brent Montague", email = "brent@bmontague.com" },
```